docs/markdown/Pkgconfig-module.md

```diff
@@ -51,10 +51,19 @@ keyword arguments.
 - `version` a string describing the version of this library
 - `d_module_versions` a list of module version flags used when compiling
    D sources referred to by this pkg-config file
+- `uninstalled_variables` used instead of the `variables` keyword argument, when
+  generating the uninstalled pkg-config file. Since *0.49.0*
 
 Since 0.46 a `StaticLibrary` or `SharedLibrary` object can optionally be passed
 as first positional argument. If one is provided a default value will be
 provided for all required fields of the pc file:
 - `install_dir` is set to `pkgconfig` folder in the same location than the provided library.
 - `description` is set to the project's name followed by the library's name.
 - `name` is set to the library's name.
+
+Since 0.49.0 uninstalled pkg-config files are generated as well. They are
+located in `<build dir>/pkgconfig-uninstalled/*-uninstalled.pc`. It is useful to
+build projects against libraries built by meson without having to install them
+into a prefix. In order to do so, just set `PKG_CONFIG_PATH=<builddir>/pkgconfig-uninstalled`
+before building your application. That will cause pkg-config to use those pc files
+in priority and find libraries and headers from the meson builddir.
```

Comment:
'pkg-config will prefer the `-uninstalled.pc` files, for compilation/linking against...'